scripts.js

```diff
@@ -22,41 +22,63 @@ hamburger.addEventListener("click", toggleMenu);
  Modal
  */
 
+const openModal = document.querySelectorAll(".open-modal");
+const closeModal = document.querySelector(".btn-close");
 
- const openModal = document.querySelectorAll(".open-modal");
- const closeModal = document.querySelector(".btn-close");
-
- function toggleModal () {
-  if (openModal){
-    document.getElementById('id1').style.display='block';
-  } else{
-    document.getElementById('id1').style.display='none';
+function toggleModal() {
+  if (openModal) {
+    document.getElementById("id1").style.display = "block";
+  } else {
+    document.getElementById("id1").style.display = "none";
   }
-  
- }
+}
 
- openModal.forEach( 
-  function(button) { 
-    button.addEventListener("click", toggleModal);
-  }
-)
+openModal.forEach(function (button) {
+  button.addEventListener("click", toggleModal);
+});
 
 closeModal.addEventListener("click", hideModal);
-function hideModal (){
-  document.getElementById('id1').style.display='none';
+function hideModal() {
+  document.getElementById("id1").style.display = "none";
 }
 
 /**
  Validate form
  */
 
+/**
+ Variables to store form elements
+ */
+const submitButton = document.getElementById("send-button");
+submitButton.addEventListener("click", formSubmit);
 
-document.getElementById("contact-form").addEventListener('submit',
-(event) =>{
-
+function formSubmit(event) {
+  const userName = document.getElementById("fullname").value;
   const email = document.getElementById("email").value;
-  if (email !== email.toLowerCase()){
-    event.preventDefault();
-    document.getElementById("error").classList.add('show');
+  const message = document.getElementById("msg").value;
+  event.preventDefault();
+  if (email !== email.toLowerCase()) {
+    document.getElementById("error").classList.add("show");
+  } else {
+    const userData = JSON.stringify({
+      userName: userName,
+      email: email,
+      message: message,
+    });
+    window.localStorage.setItem("userData", userData);
+    document.getElementById("contact-form").reset();
   }
-});
\ No newline at end of file
+}                                                                      
+window.onload = function () {
+  readLocalStorage();
+};
+/**Binding */
+
+function readLocalStorage() {
+  if (window.localStorage.getItem("userData")) {
+    const userData = JSON.parse(window.localStorage.getItem("userData"));
+    document.getElementById("fullname").value = userData.userName;
+    document.getElementById("email").value = userData.email;
```

Comment:
- [ ] Good job working on preserving the data in the local storage when the submit button is clicked. However, kindly make sure that after filling out the form, and not submitting it, the entered values should be saved in the browser.